### PR TITLE
Krishna/change deadline command argument

### PR DESCRIPTION
**Pull request by krishnaindani**
- Use flag.Duration instead of using uint. In the description, I have added what all is supported by the ParseDuration method which is a parser to flag.Duration. What do you think about it?

Issue: https://github.com/connorkuehl/popple/issues/42

### CODE REVIEW

File: popple.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func main() {
 	numWorkers := flag.Uint("workers", DefaultWorkers, "Number of worker threads to spawn")
 	dbFile := flag.String("db", Database, "Path to database file")
 	numJobs := flag.Uint("jobs", DefaultJobs, "Maximum queue size for jobs")
-	timeout := flag.Uint("deadline", 3, "seconds to wait for workers to exit when shutting down")
+	timeout := flag.Duration("deadline", 3*time.Second, "Duration to wait for workers to exit when shutting down. Its accepts decimal number with unit suffix such as \"300ms\", \"3s\". Valid time units are \"ns\", \"us\" (or \"µs\"), \"ms\", \"s\", \"m\", \"h\".")
```
Comment (connorkuehl) [inline popple.go:40]:
```suggestion
	timeout := flag.Duration("deadline", 3*time.Second, "How long to wait for workers to exit when shutting down")
Write
```

**Review by connorkuehl — CHANGES_REQUESTED** [review]
Thanks, @krishnaindani! This looks good. I think the help text is a little long, so I've suggested something a little more brief.

File: popple.go (hunk 1 above)
Comment (krishnaindani) [reply popple.go:40]: do you think the user will know how to pass the timeout based on the convention that is parsable by the function? LIke with description I don't know how the user will know to pass in seconds or minutes. Correct me if I am missing anything.

**Comment by connorkuehl** [comment]
I appreciate that perspective; however, that level of detail might be
better served being described in the README.md. You don't have to add
that as part of this PR.

I am likely the only person that runs this bot, and I know what the
switch does, so :-)

**Comment by krishnaindani** [comment]
> I appreciate that perspective; however, that level of detail might be better served being described in the README.md. You don't have to add that as part of this PR. I am likely the only person that runs this bot, and I know what the switch does, so :-)

yeah, that completely makes sense adding a part of the readme. I will push the changes.

**Comment by connorkuehl** [comment]
This looks great, thank you!

**Review by connorkuehl — APPROVED** [review]

**Comment by krishnaindani** [comment]
Thank you so much for the opportunity to contribute.

**Pull request merged**